src/components/Nav/Nav.jsx

```diff
@@ -1,11 +1,70 @@
+import {useRef} from 'react'
+import { NavLink } from 'react-router-dom'
 import './Nav.scss'
 
 const Nav = () => {
+
+    const navRef = useRef();
+
+    const showNavbar = () => {
+        navRef.current.classList.toggle('responsive_nav')
+        // Scrolling to top of the page
+        window.scrollTo({
+            top: 0,
+          });
+    }
+
     return (
-        <>
-            
-        </>
-    )
+      <header className="header">
+        <nav className="nav-container" ref={navRef}>
+            <ul className="nav-list">
+                <li className="nav-list__element">
+                    <NavLink
+                        to
+                        className={({ isActive }) => (isActive ? 'active' : '')}
+                        onClick={showNavbar}
+                    >o nas
+                    </NavLink>
+                </li>
+                <li className="nav-list__element">
+                    <NavLink
+                        to
+                        className={({ isActive }) => (isActive ? 'active' : '')}
+                        onClick={showNavbar}
+                    >oferta
+                    </NavLink>
+                </li>
+                <li className="nav-list__element">
+                    <NavLink
+                        to
+                        className={({ isActive }) => (isActive ? 'active' : '')}
+                        onClick={showNavbar}
+                    >succes story
+                    </NavLink>
+                </li>
+                <li className="nav-list__element">
+                    <NavLink
+                        to
+                        className={({ isActive }) => (isActive ? 'active' : '')}
+                        onClick={showNavbar}
+                    >blog
+                    </NavLink>
+                </li>
+                <li className="nav-list__element">
+                    <NavLink
+                        to
+                        className={({ isActive }) => (isActive ? 'active' : '')}
+                        onClick={showNavbar}
+                    >kontakt
+                    </NavLink>
```

Comment:
spokojnie możesz przemapować tutaj, np.:
```suggestion
const navItems = ["o nas", "oferta", ...]
                navItems.map((item, index) => (
                <li className="nav-list__element">
                    <NavLink
                        to
                        className={({ isActive }) => (isActive ? 'active' : '')}
                        onClick={showNavbar}
                        key={index}
                    >{item}
                    </NavLink>
                </li>
                ))
            
```